SU-SWS/decanter
# 436 437 cards

# READY FOR REVIEW

# Summary
- #436 Add background color to card contents
- #437 Add `.su-card--minimal` variant

# Needed By (Date)
- End of week

# Urgency
- Useful for #435 icon variants for cards

# Steps to Test

1. Pull the branch
1. Rebuild assets
1. Rebuild style guide
1. Check out the new `.su-card--minimal` variant of a card
1. Use the inspector to add `background-color: cornsilk;` to the `.kss-modifier__wrapper` that contains all the card variants.
    - Note that vertical and horizontal cards have a white background for the card contents.
    - Note that `.su-card--minimal` does not have a background color.

# Affected Projects or Products
- Useful for #435 icon variants for cards

# Associated Issues and/or People
- #435 icon variants for cards

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
